scRNA-seq/02-filtering_scRNA.Rmd

```diff
@@ -88,13 +88,20 @@ Using the `coldata` data frame that we set up, we can now run the `tximeta()` to
 The first time you run `tximeta()` you may get a message about storing downloaded transcriptome data in a cache directory so that it can retrieve the data more quickly the next time.
 We recommend you use the cache, and accept the default location.
 
+
 ```{r read_data, live = TRUE}
 # Read in alevin results with tximeta
 bladder_sce <- tximeta(coldata, type = "alevin")
 ```
 
+A quick aside!
+When we ran `alevinQC` on this data in the last notebook, we saw that `salmon` had identified a "whitelist" of barcodes, i.e. barcodes that passed its quality control standards.
+Here, we are actually going to read in all barcodes, regardless of whether they are on the whitelist, so that we can explore different filtering thresholds ourselves (`salmon` can be very stringent when filtering out barcodes).
+But, if you wanted to read in the data considering only barcodes which passed `salmon`'s filter, you can supply the additional argument `alevinArgs = list(filterBarcodes=TRUE)` to the `tximeta()` function. 
+Even if you do choose to read in pre-filtered data approach, it's still important to explore the data as we're about to do here and potentially filter further based on your observations, in particular since mapping software's quality control measures (spoilers!) don't always filter based on mitochondrial gene content. 
```

Comment:
```suggestion
Even if you do choose to read in pre-filtered data, it's still important to explore the data as we're about to do here and potentially filter further based on your observations, in particular since mapping software's quality control measures (spoilers!) don't always filter based on mitochondrial gene content. 
```